app/scripts/components/analysis/page-hero-media.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useEffect, useRef, useState } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { rgba, themeVal } from '@devseed-ui/theme-provider';
+import mapboxgl from 'mapbox-gl';
+import { Feature, MultiPolygon } from 'geojson';
+import bbox from '@turf/bbox';
+import { shade } from 'polished';
+
+import { SimpleMap } from '$components/common/mapbox/map';
+
+const WORLD_POLYGON = [
+  [180, 90],
+  [-180, 90],
+  [-180, -90],
+  [180, -90],
+  [180, 90]
+];
+
+const mapOptions: Partial<mapboxgl.MapboxOptions> = {
+  style: process.env.MAPBOX_STYLE_URL,
+  logoPosition: 'bottom-right',
+  interactive: false
+};
+
+interface PageHeroMediaProps {
+  feature: Feature<MultiPolygon>;
+}
+
+function PageHeroMedia(props: PageHeroMediaProps) {
+  const { feature, ...rest } = props;
+  const mapContainer = useRef<HTMLDivElement>(null);
+  const mapRef = useRef<mapboxgl.Map>(null);
+  const [isMapLoaded, setMapLoaded] = useState(false);
+
+  const theme = useTheme();
+
+  useEffect(() => {
+    if (!isMapLoaded || !mapRef.current) return;
+
+    const aoiSource = mapRef.current?.getSource('aoi');
+
+    // Quick copy.
+    const featureInverse = JSON.parse(
+      JSON.stringify(feature)
+    ) as Feature<MultiPolygon>;
```

Comment:
short answer: to make a copy of the object.

We need to add the world polygon to the coordinates array, so instead of spreading several levels of object, this was quicker. 🙃 Open to another option.

Reply:
Aah, ok, got it. One alternative could be using lodash's clone Deep (pro: more explicit, con: using a dependency)
Speaking of lodash: https://github.com/NASA-IMPACT/delta-ui/pull/284/commits/26f22d8466f707fda3d420f6949c9897fbcd55b1